webapp/.env.example

```diff
@@ -2,4 +2,6 @@ ALLOW_INDEXING=false
 DATABASE_URL=postgresql://user:password@localhost:5432/webapp
 LOG_LEVEL=info
 NODE_ENV=development
-WALLET_CONNECT_PROJECT_ID=placeholder
+WALLET_CONNECT_PROJECT_ID=:placeholder:
+L1_RPC_CLI=":placeholder:"
+ETH_NETWORK="mainnet"
```

Comment:
Added env variables must also be added to `webapp/config/env.server.ts`. This file allows us to always start the app with a validated environment. Env variables should also be imported from here.